common/lcs/tests/serde.rs

```diff
@@ -4,7 +4,7 @@
 // For some reason deriving `Arbitrary` results in clippy firing a `unit_arg` violation
 #![allow(clippy::unit_arg)]
 
-use libra_canonical_serialization::{from_bytes, to_bytes, Error, MAX_SEQUENCE_LENGTH};
+use libra_canonical_serialization::{from_bytes, to_bytes, trace::*, Error, MAX_SEQUENCE_LENGTH};
```

Comment:
Should we  re-export the symbols in `trace::prelude` and `use trace::prelude::*` instead?

Reply:
Thanks. I removed the `::*` in a couple of places.